lib/services/monitor/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
How is the client in this folder "lib/services/monitor" different than the client over [here](https://github.com/Azure/azure-sdk-for-node/tree/master/lib/services/monitoring)